packages/medusa/.npmignore

```diff
@@ -7,3 +7,4 @@ src
 ormconfig.json
 tsconfig.json
 jest.config.md
+*.tgz
```

Comment:
i can remove it, but thought in case someone accidentally left binaries like tgz after an npm pack, and pushed code in future, the system will automatically ignore it.